index.bs

```diff
@@ -1627,6 +1664,39 @@ Releases the resources associated with the {{MLTensor}}. This method is idempote
 
 Note: Since no further operations can be enqueued using this tensor, implementations can free any additional resource allocations associated with this tensor once all previously submitted operations using it are complete.
 
+## {{MLConstantTensor}} interface ## {#api-mlconstanttensor}
+
+The {{MLConstantTensor}} interface represents a tensor which may be used as a constant to an {{MLGraph}}. The memory backing an {{MLConstantTensor}} should be allocated in an [=implementation-defined=] fashion according to the requirements of the {{MLContext}} and the {{MLOperandDescriptor}} used to create it. Operations involving the {{MLTensor/[[data]]}} of an {{MLConstantTensor}} occur on the {{MLContext/[[timeline]]}} of its associated {{MLContext}}.
```

Comment:
Following on discussion (which is on outdated change): https://github.com/webmachinelearning/webnn/pull/830#discussion_r2004938741.

Given operations on MLConstantTensor.data should occur on the MLContext.timeline, the [MLGraphBuilder.build()](https://pr-preview.s3.amazonaws.com/bbernhar/webnn/pull/830.html#dom-mlgraphbuilder-build) steps may need to reflect that, because it now may access the MLConstantTensor.data.

Reply:
Yes, good point. Not 100% sure on the formal language: seems "in parallel" should become "enqueue...". PTAL again.